src/audiomixerboard.cpp

```diff
@@ -40,6 +40,8 @@ CChannelFader::CChannelFader ( QWidget*     pNW,
     plbrChannelLevel            = new CMultiColorLEDBar ( pLevelsBox );
     pFader                      = new QSlider           ( Qt::Vertical, pLevelsBox );
 
+    pPan						= new QSlider			(Qt::Horizontal, pLevelsBox);
```

Comment:
spaces and tabs should be consistent with the existing code

Reply:
Thanks a lot for the reviews. I comment below and please let me know what you prefer -  if I make the corrections as much I can and make a new pull request (most likely you need to put your hands on it on some points anyway) or you prefer to handel the code yourself.

Tabs and spacing -  sure. Sorry, I had tabs for identation first and then after change corrected as many places I found but several probably slipped through. Will correct that, of course.